src/map.cpp

```diff
@@ -432,7 +432,7 @@ void Map::getSpectators(SpectatorVec& spectators, const Position& centerPos, boo
 					}
 				} else {
 					const SpectatorVec& cachedSpectators = it->second;
-					for (Creature* spectator : cachedSpectators) {
+					for (auto spectator : cachedSpectators) {
```

Comment:
Could this be `const auto&`
```suggestion
					for (const auto& spectator : cachedSpectators) {
```